cardano-testnet/src/Testnet/Start/Types.hs

```diff
@@ -102,25 +104,21 @@ data TopologyType
 instance Default TopologyType where
   def = DirectTopology
 
-data CreateEnvUpdateTime
-  = CreateEnv
-  | UpdateTimeAndExit
+newtype UpdateTimestamps = UpdateTimestamps Bool
```

Comment:
Why a `newtype` and not a sum type? I think a sum type may be a little bit more self-explanatory.